stack.yaml

```diff
@@ -0,0 +1,76 @@
+packages:
+- location:
+    git: git@github.com:haskell-distributed/network-transport
+    commit: master
+- location:
+    git: git@github.com:haskell-distributed/network-transport-composed
+    commit: master
+# disabled due to the failing tests
+#- location:
+#    git: git@github.com:haskell-distributed/network-transport-inmemory
+#    commit: 45937f8
+- location:
+    git: git@github.com:haskell-distributed/network-transport-tcp
+    commit: master
+- location:
+    git: git@github.com:haskell-distributed/network-transport-tests
+    commit: master
+- location:
+    git: git@github.com:haskell-distributed/rank1dynamic
+    commit: master
+- location:
+    git: git@github.com:haskell-distributed/distributed-static
+    commit: master
+- location:
+    git: git@github.com:haskell-distributed/distributed-process
+    commit: master
+#- location:
+#    git: git@github.com:haskell-distributed/distributed-process-platform
+#    commit: master
+- location:
+    #git: git@github.com:haskell-distributed/distributed-process-extras
+    git: git@github.com:qnikst/distributed-process-extras
```

Comment:
why the location change? Let's get this merged once upstream changes are merged.